src/helpers/userAnalyticHelpers.js

```diff
@@ -1,7 +1,41 @@
+// Obtain country from IP address
+export function countryLookup() {
+  // Determine the country the user is from, based on IP
+  // Geoapify offers 3000 lookups/day, so we should be fine
+  var apiKey = process.env.REACT_APP_GEOAPIFY_API_KEY;
```

Comment:
can you refactor this into a static class method inside of `src/helpers/Environment.ts`? We've been isolating our `process.env` calls inside of that class.

Reply:
Done.